backend/Dockerfile

```diff
@@ -5,6 +5,7 @@ WORKDIR '/app'
 COPY package.json package-lock.json ./
 RUN npm install --production
 COPY . .
+RUN npm run build
```

Comment:
Was this intentional? I don't see a `build` script in `package.json`.

Reply:
had to do this  to make the app works locally

Reply:
hmm yeah i don't think this is needed indeed since there's no build step for the backend